Makefile

```
@@ -10,8 +10,7 @@ help:
 	@echo "\tbuild-fetcher-data-rs \t\t\tRealiza o build da imagem com as dependência do fetcher do tá na mesa"
 	@echo "\tfetch-data-rs ano=<ano> \t\tExecuta a cli do fetcher para o ano passado como parâmetro. (2019 é o default)"
 	@echo "\tfetch-data-pe ano_inicial=<ano> ano_final=<ano> \t\t\tRecupera dados do TCE-PE."
-	@echo "\tfetch-data-federal \t\t\tExecuta cli do fetcher dos dados do Governo Federal, podendo escolher qual dados específico deve ser baixado"
-	@echo "\tfetch-data-federal-all \t\t\tRecupera todos os dados do Governo Federal presentes no drive da Transparência Brasil"
+	@echo "\tfetch-data-federal \t\t\tBaixa os dados federais usados no repositório de acordo com um intervalo de tempo. Ex: make fetch_data_federal data_inicio=2020-06-19 data_fim-2020-06-20"
```

```suggestion
	@echo "\tfetch-data-federal \t\t\tBaixa os dados federais usados no repositório de acordo com um intervalo de tempo. Ex: make fetch_data_federal data_inicio=2020-06-19 data_fim=2020-06-20"
```
Erro de digitação no parâmetro do make